src/main/java/org/elmlang/intellijplugin/Elm.flex

```diff
@@ -59,6 +60,26 @@ RESERVED=("hiding" | "export" | "foreign" | "deriving")
     }
 }
 
+<IN_DOC_COMMENT> {
```

Comment:
doc comments don't need their own state. They should share the state with level comments because they are in fact level comments that just start with a `|` character as a first character of the comment.